Gemfile

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
+
+ruby "2.7.2"
```

Comment:
i assume this was the main issue?

Reply:
Apparently not. But it did expose the fact that Travis isn't actually honoring the `.ruby-version` file.

Reply:
Good to know.